ios/MarigoldSDKReactNativeTests/RNMarigoldBridgeTests.m

```diff
@@ -0,0 +1,122 @@
+#import <XCTest/XCTest.h>
+#import "RNMarigoldBridge.h"
+#import "RNMarigold.h"
+#import "RNEngageBySailthru.h"
+#import "Kiwi.h"
+
+@interface Marigold ()
+
+- (void)setWrapperName:(NSString *)wrapperName andVersion:(NSString *)wrapperVersion;
+
+@end
+
+SPEC_BEGIN(RNMarigoldBridgeSpec)
+
+describe(@"RNMarigoldBridge", ^{
+    __block Marigold *marigold = nil;
```

Comment:
This is no longer present in the class so you should be able to remover it and all the related stubbing.